src/hound/searcher/searcher.go

```diff
@@ -133,6 +133,7 @@ func New(vcsDir string, repo *config.Repo) (*Searcher, error) {
 	log.Printf("Searcher started for %s", name)
 
 	sha, err := vcs.PullOrClone(repo.VCS, vcsDir, repo.Url)
+
```

Comment:
This looks like an empty change - can you just pull this out since this file doesn't have any changes otherwise?